modules/identity-java-agent/java-agent/src/main/java/org/wso2/carbon/identity/java/agent/config/InterceptorConfigReader.java

```diff
@@ -18,32 +18,58 @@
 
 package org.wso2.carbon.identity.java.agent.config;
 
+
+
+import java.io.FileInputStream;
+import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Properties;
 
 /**
  * Reads the interceptor config from the resources file in the classpath.
  */
 public class InterceptorConfigReader {
+    public static final String filename = "/home/piyumi/identity-developer-tools/modules/identity-java-agent/java-agent/src/main/resources/instrumentation-config.properties";
+    private String className;
+    private String methodName;
+    private String signature;
 
     /**
      * Reads the configs in the class resource
-     * "instrumentation-config.json".
+     * "instrumentation-config.properties".
      *
      * @return
      */
-    public List<InterceptorConfig> readConfig() {
+    public List<InterceptorConfig> readConfig(){
+
+
+            Properties properties = new Properties();
+            try {
+                FileInputStream fileReader = new FileInputStream(filename);
```

Comment:
Will need to change to read any number of properties.
Use buffered reader, and read line. then parse each line to create and populate InterceptorConfig object